docs/user/vulnerabilities/vulnerability/product_status/known_affected-usage-csaf_vex.en.md

```diff
@@ -1,3 +1,3 @@
 # Known Affected - Usage
 
-* For the `csaf_vex` profile, for each item in `known_affected` a corresponding action statement in `/vulnerabilities[]/remediations` must exist.
+For the `csaf_vex` profile, for each item in `known_affected` a corresponding action statement in `/vulnerabilities[]/remediations` must exist.
```

Comment:
Add:
The action statement describes what a user should do. Optional, additional information MAY also be provide through `/vulnerabilities[]/notes` and `/vulnerabilities[]/threats`.